src/lib/remembergaza_web/live/home_live.html.heex

```diff
@@ -0,0 +1,37 @@
+<center class="flex flex-col py-6 px-6 md:py-12 md:px-12">
+  <h1 class="mb-8">Remember Gaza</h1>
+
+  <p class="mb-8">
+    There are <strong><%= @total_identified_victims %></strong> identified
+    victims killed in Gaza since October 7, 2023.
+    <br />
+    While the remaining <strong><%= @unidentified_victims %></strong> victims
+    are still unidentified.
+  </p>
+
+  <p class="mb-8">
+    Total souls killed in Gaza: <strong><%= @total_victims %></strong>.
+  </p>
+
+  <div
+    id="victims"
+    phx-update="stream"
+    phx-viewport-top={@page > 1 && "prev-page"}
+    phx-viewport-bottom="next-page"
+    phx-page-loading
+    class="grid gap-6 md:grid-cols-3 mb-8"
+  >
+    <div :for={{_, victim} <- @streams.victims}>
+      <h3><%= victim["name"] %></h3>
+      <p><%= victim["en_name"] %></p>
+      <%= if victim["age"] != nil and victim["age"] != -1 do %>
+        <p class="text-sm text-slate-500"><%= victim["age"] %> years old</p>
+      <% end %>
```

Comment:
I've ported your changes here @AsadSaleh
The difference is we do not display the "unknown age" anymore:

<img width="480" alt="image" src="https://github.com/ariefrahmansyah/remembergaza/assets/8122852/fee68a90-acbd-4ec8-875c-f12ec72f82ee">

Reply:
Hmm.., will there be a height inconsistency because of it?

Reply:
After looking at it again, I think the height is all consistent.